script.js

```diff
@@ -0,0 +1,89 @@
+
+var tiles = document.querySelectorAll(".tile")
+
+var moves = 0
+
+
+function tilesClicked(evt) {
+	// console.log(evt);
+	var tile = evt.target
+	console.log(evt)
+	if(tile.innerHTML === '') {
+		moves = moves + 1;
+		if(moves % 2 === 0) {
+			tile.innerHTML = "o";
+			tile.className += " odd";
+		}
+		else {
+			tile.innerHTML = "x";
+			tile.className += " even";
+		}
+	}
+}
+
+for (var i = 0; i < tiles.length; i++) {
+	tiles[i].addEventListener('click', tilesClicked);
+}
```

Comment:
Awesome job making a function for the tile click behavior, as well as using a loop in order to assign the event listeners.